formulas/dnsmasq/README.md

```diff
@@ -13,7 +15,7 @@ Examples
 
     wickFormula dnsmasq --start
 
-Returns nothing
+    wickFormula dnsmasq --protect-dns
```

Comment:
I don't see the line in the run script.  I only see it in the README.md, which should be entirely generated from the run script.  :-(